src/engine/outfit.ts

```diff
@@ -141,16 +142,27 @@ export function baseOutfit(allowAttackingFamiliars = true): OutfitSpec {
   const lovTunnelCompleted = get("_loveTunnelUsed") || !get("loveTunnelAvailable");
 
   return {
+    weapon: myPrimestat() === $stat`Muscle` && have($item`june cleaver`)
+      ? $item`june cleaver`
+      : have($item`fish hatchet`)
+      ? $item`fish hatchet`
+      : have($item`bass clarinet`)
+      ? $item`bass clarinet`
+      : undefined,
```

Comment:
Why are we choosing a default weapon here instead of leaving it to the maximizer to decide? Specifically, why are we running around with a June Cleaver instead of a `mainStat%` weapon by default?

Reply:
When scaling muscle, scaling monster HP scales higher. We need to prioritize damage as a result. This is the fix for going past round 30 in some later fights for Muscle classes.

Reply:
This doesn't seem right then. It should be
```
myPrimestat() === $stat`Muscle` ? (weapon logic) : undefined
```

What you have here would force equip the hatchet on a non-muscle class

Reply:
Changed the order on this a bit. If you grab fish hatchet, you should use fish hatchet. If you're saving floundry, you won't have it. In that scenario, if you're muscle class and you have Cleaver, you want cleaver.

I lowered Cleaver in ranking here to reflect that.